# Identifying biomarkers in adult febrile patients of malaria and COVID-19 using RNA-seq data


![image](https://github.com/omicscodeathon/circrna_biomarker/blob/main/figures/bckg_11.jpg)
## Background


The pathogenesis of infectious diseases depends on the interaction of host and pathogen.
Patients with malaria always present overlapping clinical presentation like COVID-19 infection.
Many febrile patients are misdiagnosed due to the paucity of seroprevalence data malaria and COVID-19 coinfection.
RNA-seq provides a sensitive method for global gene expression analysis in infection disease.
![image](https://user-images.githubusercontent.com/67194450/160154386-0562c21a-e0f3-4a5d-9c81-341658ff6644.png)

## Problem statment
Differentiate the etiology of febrile patient if is either malaria or COVID-19.

## Objective of the study.
 We will examined the immune system responses of patient infected with malaria and COVID-19 to identify a unique biomarker signature in both infections.
### Specific objectives.
 1. Comparing expression patterns of mono infection and co-infection.
 2. Determine differentially expressed genes and synergistic pathways involved.

## Hypothesis
We hypothesize that the transcriptomic profiles from febrile patient with malaria  and COVID-19 are associated with the overlapping symptoms. 

## Methodology | Overall flow of the experimental design
![image](https://github.com/omicscodeathon/circrna_biomarker/blob/main/figures/wk.png)


## Our Target
To improve our understanding in immne responses for a patient with malaria and COVID-19 as coinfection.
We undertook transcriptomics data to identify gene expression patterns within a patient with 
either malaria or COVID-19.



## Workflow 
![image](https://github.com/omicscodeathon/circrna_biomarker/blob/main/figures/workflow_1.png)



## Results
Quality control of dataset (i.e Quality control of one COVID-19 patient). 
![image](https://github.com/omicscodeathon/circrna_biomarker/blob/main/figures/QC-COVID19_p1.png)

##### 

## Team 
1. NZUNGIZE Lambert, Data collection and Analysis, Email: nzulapa@outlook.com, CRID/LSTM, Cameroon.
2. Brenda Muthoni, Data collection and Analysis
3. Arnaud Kengne-Ouafo Jonas, Data collection and Analysis, Email: jakengne@gmail.com, CRID/LSTM, Cameroon.
4. kivumbi Mark Tefero, Github content, Email: markkivumbi@gmail.com, Pwani University, Kenya.
5. Margaret Wanjiku, Manuscript writing
6. Brenda Kiage, Manuscript writing, Email: kiagebrenda@gmail.com, Pwani University, Kenya.
7. UMUHOZA Diane, Manuscript writing, Email: ddumuhoza@gmail.com, RAB-Rwanda.
